tests/extensions/extra/tables.txt

```diff
@@ -80,3 +80,13 @@ Lists are not tables
 
  - this | should | not
  - be | a | table
+
+Add tests for issue #449
+
+Odd backtics | Even backtics
```

Comment:
Wasn't sure how that affects the pull as I think GitHub uses the message to calculate the hash.  Good to know it can be done.  I'll take care of it.